.gitignore

```diff
@@ -1,5 +1,3 @@
-config.env
```

Comment:
I think that this line have to be replaced "config.env" with ".env" instead of the removal.

Reply:
What? `.env` is already in the content after.

Reply:
What? `.env` is already in the content after.

Reply:
Yep, it's down near the bottom of the file.